core/tests/puppeteer-acceptance-tests/specs/blog-editor/create-and-delete-draft-blog-post.spec.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for creating and deleting draft blog posts.
+ */
+
+import {UserFactory} from '../../utilities/common/user-factory';
+import testConstants from '../../utilities/common/test-constants';
+import {BlogPostEditor} from '../../utilities/user/blog-post-editor';
+
+const DEFAULT_SPEC_TIMEOUT_MSECS = testConstants.DEFAULT_SPEC_TIMEOUT_MSECS;
+const ROLES = testConstants.Roles;
+
+describe('Blog Editor', function () {
+  let blogPostEditor: BlogPostEditor;
+
+  beforeAll(async function () {
+    blogPostEditor = await UserFactory.createNewUser(
+      'blogPostEditor',
+      'blog_post_editor@example.com',
+      [ROLES.BLOG_POST_EDITOR]
+    );
+  }, DEFAULT_SPEC_TIMEOUT_MSECS);
```

Comment:
Are we manually configuring this everywhere? (timeout secs)

Isn't it possible to configure it globally?

Reply:
Currently, we are doing in this way only.

Reply:
@rahat2134 Can you please reply to the above question?

Reply:
> Currently, we are doing in this way only.

Considering you are going to write many more test, can you please check if we can configure this globally and avoid having to configure every time?